From a119a7ee62912bcc438093fcbff59131f7f76098 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 22 Oct 2012 15:04:23 +0800
Subject: [PATCH 081/602] ENGR00230588 Add build_id.mk to config the BUILD_ID

BUILD_ID need to be updated based on each board's build_id.mk

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/BoardConfig.mk     |  1 +
 arm2_6dq/build_id.mk        | 21 +++++++++++++++++++++
 arm2_6sl/BoardConfig.mk     |  1 +
 arm2_6sl/build_id.mk        | 21 +++++++++++++++++++++
 sabreauto_6q/BoardConfig.mk |  1 +
 sabreauto_6q/build_id.mk    | 21 +++++++++++++++++++++
 sabresd_6dq/BoardConfig.mk  |  1 +
 sabresd_6dq/build_id.mk     | 21 +++++++++++++++++++++
 8 files changed, 88 insertions(+)
 create mode 100644 arm2_6dq/build_id.mk
 create mode 100644 arm2_6sl/build_id.mk
 create mode 100644 sabreauto_6q/build_id.mk
 create mode 100644 sabresd_6dq/build_id.mk

diff --git a/arm2_6dq/BoardConfig.mk b/arm2_6dq/BoardConfig.mk
index 5ad7d1f..b04aa19 100755
--- a/arm2_6dq/BoardConfig.mk
+++ b/arm2_6dq/BoardConfig.mk
@@ -3,6 +3,7 @@
 #
 
 include device/fsl/imx6/soc/imx6dq.mk
+include device/fsl/arm2_6dq/build_id.mk
 include device/fsl/imx6/arm2/BoardConfigArm2.mk
 include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
new file mode 100644
index 0000000..f2a2451
--- /dev/null
+++ b/arm2_6dq/build_id.mk
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2008 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# BUILD_ID is usually used to specify the branch name
+# (like "MAIN") or a branch name and a release candidate
+# (like "CRB01").  It must be a single word, and is
+# capitalized by convention.
+
+export BUILD_ID=R15-Alpha
diff --git a/arm2_6sl/BoardConfig.mk b/arm2_6sl/BoardConfig.mk
index 917d069..0498611 100755
--- a/arm2_6sl/BoardConfig.mk
+++ b/arm2_6sl/BoardConfig.mk
@@ -3,6 +3,7 @@
 #
 
 include device/fsl/imx6/soc/imx6sl.mk
+include device/fsl/arm2_6sl/build_id.mk
 include device/fsl/imx6/arm2/BoardConfigArm2.mk
 include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
new file mode 100644
index 0000000..f2a2451
--- /dev/null
+++ b/arm2_6sl/build_id.mk
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2008 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# BUILD_ID is usually used to specify the branch name
+# (like "MAIN") or a branch name and a release candidate
+# (like "CRB01").  It must be a single word, and is
+# capitalized by convention.
+
+export BUILD_ID=R15-Alpha
diff --git a/sabreauto_6q/BoardConfig.mk b/sabreauto_6q/BoardConfig.mk
index edef186..a5fe5b5 100644
--- a/sabreauto_6q/BoardConfig.mk
+++ b/sabreauto_6q/BoardConfig.mk
@@ -2,6 +2,7 @@
 # Product-specific compile-time definitions.
 #
 
+include device/fsl/sabreauto_6q/build_id.mk
 include device/fsl/imx6/BoardConfigCommon.mk
 
 TARGET_BOOTLOADER_BOARD_NAME := SABREAUTO
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
new file mode 100644
index 0000000..f2a2451
--- /dev/null
+++ b/sabreauto_6q/build_id.mk
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2008 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# BUILD_ID is usually used to specify the branch name
+# (like "MAIN") or a branch name and a release candidate
+# (like "CRB01").  It must be a single word, and is
+# capitalized by convention.
+
+export BUILD_ID=R15-Alpha
diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 68a4807..98597dd 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -3,6 +3,7 @@
 #
 
 include device/fsl/imx6/soc/imx6dq.mk
+include device/fsl/sabresd_6dq/build_id.mk
 include device/fsl/imx6/sabresd/SabreSDBoardConfigComm.mk
 include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
new file mode 100644
index 0000000..f2a2451
--- /dev/null
+++ b/sabresd_6dq/build_id.mk
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2008 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# BUILD_ID is usually used to specify the branch name
+# (like "MAIN") or a branch name and a release candidate
+# (like "CRB01").  It must be a single word, and is
+# capitalized by convention.
+
+export BUILD_ID=R15-Alpha
-- 
1.8.0

